add HTTP inspection server with Takes framework

ErnestMatskevich opened the pull request:
Add inspect command which run web server with Takes framework and allow to send text from console to server and displays response from server in console (just testing server running, without real inspecting functionality)

Add Takes dependency in pom.xml

@yegor256 Please can you take a look and answer am I correct starting implement server part for inspect command and working with maven in proper way? 

I intentionally didn't include tests in this PR as I wanted to first align on the architectural approach. I'm ready to add tests in the next pull request after getting your approval on the current implementation."

yegor256 commented:
@ErnestMatskevich I suggest fixing the build first, read this: https://www.yegor256.com/2025/04/19/dont-merge-into-broken-master.html

yegor256 commented:
@ErnestMatskevich please, add qulice to the pom.xml, to make sure Java code is clean: www.qulice.com

yegor256 commented:
@maxonfjvipon please, take a look

0crat commented:
@ernestmatskevich Hey there! 👋 We noticed your Git branch name "`inspect_server`" could be improved. For better project organization, we recommend [using ticket numbers as branch names](https://www.yegor256.com/2014/04/15/github-guidelines.html). In this case, "`521`" would be ideal. This small change helps everyone track progress more easily. You've received -12 points this time, but we're confident you'll nail it next time! 💪

<!-- It is [not a good idea](https://www.yegor256.com/2014/04/15/github-guidelines.html) to name Git branches the way you named this one: "`inspect_server`". You've earned -12 points. Next time, better give your branch the same name as the number of the ticket that you are solving. In this case, a perfect name, for example, would be "`521`".  -->

maxonfjvipon commented on src/commands/inspect.js:
```diff
@@ -0,0 +1,116 @@
+module.exports = function(opts) {
+  return new Promise((resolve, reject) => {
+    const mvnDir = path.join(process.cwd(), 'mvnw');
+
```
@ErnestMatskevich let's remove all the empty lines in this function

ErnestMatskevich replied:
Done

maxonfjvipon reviewed (changes requested):
@ErnestMatskevich good work actually, see some comments above

ErnestMatskevich commented on src/eoc.js:
```diff
@@ -447,6 +448,12 @@ program.command('fmt')
+program.command('inspect')
+  .description('Start EO inspect server and send input')
+  .action((str, opts) => {
+    coms().inspect(program.opts());
```
Added unit test in `test/commands/test_inspect.js`

ErnestMatskevich commented:
I added qulice in pom.xml and fixed Inspect.java to make its code clean, but I got such message:

```
[WARNING] Unused declared dependencies found:
1.         org.eolang:eo-runtime:jar:0.56.2:compile
2.         org.junit.jupiter:junit-jupiter-api:jar:5.12.2:compile
3.         org.junit.jupiter:junit-jupiter-engine:jar:5.12.2:compile
4.         org.takes:takes:jar:1.24.6:compile
```

@yegor256 What should I do with 1,2,3 dependencies? Seems that qulice wrong with them, because I do not use they in my Java file directly. May I add they in `<excludes>` for qulice?
Also I need help to understand why problems with Takes. I successfully use Takes in my Java file and run the server, but get warning too 

Also I would like to clarify how to rename branch name properly. 0crat suggest use 521 as a name, but after reading [Github Guidlines](https://www.yegor256.com/2014/04/15/github-guidelines.html) I uderstood that branch name should show at which issue I am currently working. Since adding an inspect command was described in issue #500 should I rename branch to this number or use 521 as was suggested by 0crat?

yegor256 commented:
@ErnestMatskevich qulice is giving you a hint: you don't have unit tests for your Java class. You are right, `500` would be a better branch name.
